src/mgmt/srv_query.c

```diff
@@ -568,3 +568,102 @@ ds_mgmt_dev_set_faulty(uuid_t dev_uuid, Mgmt__DevStateResp *resp)
 
 	return rc;
 }
+
+struct bio_replace_dev_info {
+	uuid_t		old_dev;
+	uuid_t		new_dev;
+};
+
+static int
+bio_storage_dev_replace(void *arg)
```

Comment:
It's better to remove the prefix of "bio_" from the function name.

Reply:
we use "bio_" prefix for many other functions in this file. maybe we can clean that up later?

Reply:
sure.